tests/basic/nomad_test.go

```diff
@@ -0,0 +1,415 @@
+package tests
+
+import (
+	"fmt"
+	"github.com/hashicorp/nomad/api"
+	. "github.com/onsi/ginkgo/v2"
+	"github.com/portworx/torpedo/drivers/nomad"
+	"github.com/portworx/torpedo/pkg/log"
+	. "github.com/portworx/torpedo/tests"
+	"time"
+)
+
+var _ = Describe("{ListNodesOnNomadCluster}", func() {
+	var client *nomad.NomadClient
+	var err error
+	JustBeforeEach(func() {
+		StartTorpedoTest("ListNodesOnNomadCluster", "List nodes in Nomad Cluster", nil, 0)
+		client, err = nomad.NewNomadClient()
```

Comment:
since client is being used in all nomad tests. can we move it before suite?

Reply:
Ok, good idea. Will handle this in next PR.